centos_ci.sh

```diff
@@ -272,7 +272,7 @@ function perform_release() {
   make prerelease
   exit_on_failure "$?" "Pre-release tests failed."
 
-  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags ~coolstore -format pretty"
+  MINISHIFT_VM_DRIVER=kvm make integration ~coolstore
```

Comment:
Here you should select just some tags, I would suggest `basic,openshift,version,cmd-config`, these are tags which are currently active on master branch. With ~coolstore I think you will hit a timeout on Centos-CI, we have too many tests (which is nice).

Reply:
Thanks for bringing it to notice. Will update it

Reply:
@amitkrout Any specific reason you have removed ` -format pretty`?

Reply:
@LalatenduMohanty  ```-format pretty``` is passed through Makefile as part of ```GODOG_OPTS```